src/inc/til/throttled_func.h

```diff
@@ -171,31 +183,24 @@ namespace til
         void flush()
         {
             WaitForThreadpoolTimerCallbacks(_timer.get(), true);
-            if (_storage)
-            {
-                _trailing_edge();
-            }
+            _timer_callback(nullptr, this, nullptr);
```

Comment:
Reading this code I've realized that I can ever so slightly reduce the binary size by moving `_trailing_edge` into the `_timer_callback` and calling `_timer_callback` directly.

Reply:
It depends honestly. If all of timer_callback is larger and less deduplicatable with COMDAT folding, it could be larger overall.

Reply:
Hm, I think I see what you mean. It does have the additional benefit that it swallows the exception, just like the regular timer callback. This makes it more predictable IMO, because it now behaves identical.